src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/run.py

```diff
@@ -38,7 +39,10 @@ def acr_run(cmd,
             timeout=None,
             resource_group_name=None,
             os_type=OS.linux.value):
-
+    if not file.endswith(ALLOWED_TASK_FILE_TYPES):
+        raise CLIError("Task file has an invalid suffix: {}."
+                       "It must have one of the following suffixes: {}"
+                       .format(file, ALLOWED_TASK_FILE_TYPES))
```

Comment:
Here we tell users they can pass a .toml .json as a task file. My concern is users may have the impression that they can pass any arbitrary script to run as task. Shall we limit the list to .yaml and yml? @ehotinger